core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1303,6 +1304,32 @@ public void checkAccess(AlluxioURI path, CheckAccessContext context)
     return inconsistentUris;
   }
 
+  @Override
+  public boolean checkExistence(AlluxioURI path, CheckExistenceContext context)
+      throws AccessControlException, IOException {
+    try (RpcContext rpcContext = createRpcContext(context);
+            FileSystemMasterAuditContext auditContext =
```

Comment:
indent for this and following lines
change line usually is 4 indentation

Reply:
i remember @jenoudet has a demo recording about how to configure intellij to match Alluxio codestyle. @jenoudet can you help share that to us?

Reply:
[Coding style](https://docs.alluxio.io/os/user/edge/en/contributor/Code-Conventions.html#coding-style) under `If you use IntelliJ IDEA`.

Reply:
I think the tab configuration is separate from checkstyle. Tab is configured at Code Style -> Java -> Tabs and Indents. Checkstyle didn't seem to complain about this otherwise.